### PR TITLE
cleanup release 0.13

### PR DESCRIPTION
**Pull request by andriisoldatenko**
TODO:
- [x] fix elasticsearch-exporter
- [x] fix crashing fluentd + cleanup connfig warnings

### CODE REVIEW

**Comment by andriisoldatenko** [comment]
to fix CI/CD we need to merge dependent https://github.com/astronomer/ap-vendor/pull/16

File: charts/fluentd/templates/fluentd-configmap.yaml [hunk 1]
```diff
@@ -36,7 +36,6 @@ data:
         </pattern>
       </parse>
       tag raw.kubernetes.*
-      format json
```
Comment (andriisoldatenko) [inline charts/fluentd/templates/fluentd-configmap.yaml:39]: bc unused parametr.

File: charts/fluentd/templates/fluentd-configmap.yaml [hunk 2]
```diff
@@ -124,8 +123,6 @@ data:
       key_name message
       replace_invalid_sequence true
       emit_invalid_record_to_error false
-      suppress_parse_error_log true
-      ignore_key_not_exist true
```
Comment (andriisoldatenko) [inline charts/fluentd/templates/fluentd-configmap.yaml:128]: bc unused parametr.

**Comment by andriisoldatenko** [comment]
practically it works in KinD.

image

File: charts/fluentd/values.yaml [hunk 3]
```diff
@@ -5,7 +5,7 @@ tolerations: []
 images:
   fluentd:
     repository: astronomerinc/ap-fluentd
-    tag: 1.8.0
+    tag: 1.9.2
```
Comment (sjmiller609) [inline charts/fluentd/values.yaml:8]: from Andrii's other PR https://github.com/astronomer/ap-vendor/pull/16/files

**Pull request merged**